src/cf-typography/src/atoms/links.less

```diff
@@ -109,8 +39,6 @@
 
         position: relative;
 
-        padding-right: 1.25em;
-
         &:after,
         &:before {
```

Comment:
With the new way icons are being handled, the `&:before` selector can be removed from this group.